### PR TITLE
[docs] Fix singIn method comment 

### PR DESCRIPTION
**Pull request by m98**
#### Description of changes

Parameter `password` described in JSDoc did not appear in the function signature. It needed to be `pw` as it was defined in the method itself. This PR only adds the parameter names in the method comment.

#### Checklist

- [x] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [x] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/auth/src/Auth.ts [hunk 1]
```diff
@@ -469,7 +469,8 @@ export class AuthClass {
 	/**
 	 * Sign in
 	 * @param {String | SignInOpts} usernameOrSignInOpts - The username to be signed in or the sign in options
-	 * @param {String} password - The password of the username
+	 * @param {String} pw - The password of the username
+	 * @param {ClientMetaData} clientMetadata - Client metadata
```
Comment (ashika01) [inline packages/auth/src/Auth.ts:473]:
```suggestion
	 * @param {ClientMetaData} clientMetadata - A map of custom key-value pairs for custom workflows
```

**Review by ashika01 — APPROVED** [review]

**Review by elorzafe — APPROVED** [review]
Looks good to me

Thanks @m98

**Comment by m98** [comment]
You're welcome @elorzafe
I still can't merge. Maybe you can mention another reviewer

**Review by AllanZhengYP — APPROVED** [review]

**Comment by stocaaro** [comment]
This looks good. Thank you for making this change! I'm bringing the branch up to current now and will merge this in when the tests finish running.

**Comment by stocaaro** [comment]
Merged. Thanks again for the contribution!

**Pull request merged**